test/fuzz/Big.fuzz.test.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable jest/no-standalone-expect */
+import { fuzz } from './JsonRpcFuzz2';
+
+describe('Fuzz', () => {
+  it.fuzz('should ', async (data) => {
+    await fuzz(data);
+    expect(true).toStrictEqual(true);
```

Comment:
ok so is the plan to use this as the entrypoint for the tests and format them to loop through the API spec methods with the fuzz function somehow?

Reply:
Yes, along with using the @jazzer.js/jest-runner it will pass in various, infinite amount of `Buffer` data over the time we set in the jazzer config.